File: examples/serz/serz__top.py
# serz__top
import calvin
l = calvin.Node("localhost", 9000, True)
l.load("./serz__f")
l.load("./serz__s")
serz__f = l.new("serz__f", "serz__f")
serz__s = l.new("serz__s", "serz__s")
serz__f.Out >> serz__s.In
serz__s.Out >> serz__f.In
actors = (serz__f,serz__s,)
for actor in actors:
  actor.enable()

File: examples/serz/serz__top.cs
# serz__top
LOAD ./serz__f
LOAD ./serz__s
NEW serz__f serz__f
NEW serz__s serz__s
CONNECT serz__f.Out serz__s.In
CONNECT serz__s.Out serz__f.In
ENABLE serz__f serz__s
JOIN